src/test/java/com/epam/tc/hw7/utils/MetalsColorsDataProvider.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.tc.hw7.utils;
+
+import com.epam.tc.hw7.entities.MetalsColorsFormData;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.testng.annotations.DataProvider;
+
+public class MetalsColorsDataProvider {
+
+    @DataProvider(name = "form fields")
+    public static Object[][] formData() throws FileNotFoundException {
+        ArrayList<MetalsColorsFormData> formDataList = readData();
+        return new Object[][] {
+            {formDataList.get(0)},
```

Comment:
You should not read datafile with reference to data count.
What are you going to do if you added one additional data set to the file?